assets/RNAseq_report.Rmd

```diff
@@ -1968,12 +1968,15 @@ quote_string <- ifelse(
     paste0("The results for all work packages (no quote/offer given to describe them) can be found in this report.")
 )
 cat(quote_string)
+
+# Initiate var in case no software_versions were provided
+is_csv <- F
```

Comment:
Could you give it an easier to understand name? e.g. `software_versions_csv` ?

Reply:
done!